glade_forms/lib/src/validator/int_validator.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:glade_forms/src/src.dart';
+
+class IntValidator extends GladeValidator<int> {
+  IntValidator();
+
+  /// Compares given value with [min] and [max] values. With [inclusiveInterval] set to true(default), the comparison is inclusive.
+  void isBetween({
+    required int min,
+    required int max,
+    OnValidateError<int>? devError,
+    Object? key,
+    ShouldValidateCallback<int>? shouldValidate,
+    bool inclusiveInterval = true,
+  }) =>
+      satisfy(
+        (value) => inclusiveInterval ? value >= min && value <= max : value > min && value < max,
+        devError: devError ?? (value) => 'Value ${value ?? 'NULL'} in not in between $min and $max.',
```

Comment:
Maybe also reflect the `inclusiveInterval` in the text?

Reply:
Hmm you meant in Readme.md? Because its there, otherwise I don't know what "text" stands for.

Reply:
Hope that's now ok, edited the devError message